File: Find the “Kth” max and min element of an array/sol.cpp
class Solution {
public:
    // arr : given array
    // l : starting index of the array i.e 0
    // r : ending index of the array i.e size-1
    // k : find kth smallest element and return using this function
    int kthSmallest(int arr[], int l, int r, int k) {
        std::sort(arr + l, arr + r + 1);

        int ans = 0;
        for (int i = 0; i < k; i++) {
            ans = arr[l + i];
        }

        return ans;
    }
};

File: Find the “Kth” max and min element of an array/sol.java
class Solution{
    public static int kthSmallest(int[] arr, int l, int r, int k) 
    { 
        int ans = 0;
        Arrays.sort(arr);
        for(int i=0; i<k; i++){
            ans = arr[i];
        }
        return ans;
    } 
}
